.github/workflows/create_release.yml

```diff
@@ -0,0 +1,101 @@
+name: Create Release
+
+on: [push]
+
+jobs:
+  Build-And-Test-Linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Initialize submodules
+        run: git submodule update --init --recursive
+
+      - name: Build
+        run: |
+          bazel build --config=linux --compilation_mode=opt --linkopt=-Wl,--strip-all --copt=-fdata-sections --copt=-ffunction-sections --linkopt=-Wl,--gc-sections \
```

Comment:
Are all those compiler/linker flags necessary for `-c opt`? Maybe some of them are enabled by default.

Reply:
I found they made a difference when working on the original sync, so I'm pretty sure they're not enabled by default.